src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -2,23 +2,61 @@
 
 import java.util.List;
 
+import com.iota.iri.service.API;
+
+/**
+ * 
+ * Contains information about the result of a successful {@code getNeighbors} API call.
+ * See {@link API#getNeighborsStatement} for how this response is created.
+ *
+ */
 public class GetNeighborsResponse extends AbstractResponse {
 
+    /**
+     * The neighbors you are connected with, as well as their activity count.
+     * This activity includes the following statistics:
+     * <ol>
+     *     <li>address</li>
+     *     <li>connectionType</li>
+     *     <li>numberOfAllTransactions</li>
+     *     <li>numberOfRandomTransactionRequests</li>
+     *     <li>numberOfNewTransactions</li>
+     *     <li>numberOfInvalidTransactions</li>
```

Comment:
there is now also a `getNumberOfStaleTransactions` :)
not sure it's worth maintaining a list here

Reply:
I think it is, this information is parsed by the doclet. Would be nice to see what a Neighbor actually includes in the documentation I think!